### PR TITLE
[DEV-1178] adapt to UI components

### PR DESCRIPTION
**Pull request by Krysh90**

### CODE REVIEW

File: .env.production [hunk 1]
```diff
@@ -1,2 +1,3 @@
 REACT_APP_API_URL=https://api.dfx.swiss/v1
-REACT_APP_TNC_URL=https://dfx.swiss/terms-and-conditions/
\ No newline at end of file
+REACT_APP_TNC_URL=https://dfx.swiss/terms-and-conditions/
+REACT_APP_KYC_URL=https://dev.payment.dfx.swiss/kyc
```
Comment (davidleomay) [inline .env.production:3]: https://payment.dfx.swiss/kyc

File: src/components/tabs/buy-tab-content/buy.process.tsx [hunk 2]
```diff
@@ -159,27 +163,42 @@ interface PaymentInformation {
...
+          {info.bic}
+          <StyledIconButton icon={IconVariant.COPY} onClick={() => copy(info.bic)} />
+        </StyledDataTableRow>
+        <StyledDataTableRow label="Purpose of payment">
+          {info.purpose}
+          <StyledIconButton icon={IconVariant.COPY} onClick={() => copy(info.purpose)} />
+        </StyledDataTableRow>
+      </StyledDataTable>
+      <StyledDataTable label="Recipient" darkTheme={false} showBorder>
+        <StyledDataTableRow>DFX AG, Bahnhofstrasse 7, 6300 Zug, Schweiz</StyledDataTableRow>
```
Comment (davidleomay) [inline src/components/tabs/buy-tab-content/buy.process.tsx:193]: Get the recipient from payment info?

File: .vscode/settings.json [hunk 3]
```diff
@@ -1,7 +1,18 @@
...
+    "PALEWHITE",
+    "Multichain",
+    "testid",
+    "iban",
+    "IBAN",
+    "onlineid",
+    "videoid",
+    "noreferrer",
+    "OLKILUL",
+    "Bahnhofstrasse"
```
Comment (davidleomay) [inline .vscode/settings.json:14]: These two seem to be unnecessary 🤔

File: src/components/user-data.tsx [hunk 4]
```diff
@@ -17,39 +22,40 @@ export function UserData(): JSX.Element {
...
-        func: () => user && copy(user.ref),
-      },
+      title: 'Referral code',
+      value: user?.ref ?? 'Complete a buy to receive your Ref code',
+      button:
+        user?.ref != null
+          ? {
+              color: StyledButtonColors.RED,
+              label: 'Copy to share',
+              func: () => user && copy(user.ref),
```
Comment (davidleomay) [inline src/components/user-data.tsx:51]: `user.ref` is never null

File: src/components/tabs/buy-tab-content/buy.process.tsx [hunk 5]
```diff
@@ -159,27 +163,42 @@ interface PaymentInformation {
...
+      </StyledDataTable>
+      <StyledDataTable darkTheme={false} alignContent={AlignContent.BETWEEN}>
+        <StyledDataTableRow discreet>
+          <p>DFX-Fee</p>
+          <p>{info.fee}</p>
+        </StyledDataTableRow>
+      </StyledDataTable>
+      <StyledButton label="Click once your bank Transfer is completed." onClick={onFinish} caps={false} />
```
Comment (davidleomay) [inline src/components/tabs/buy-tab-content/buy.process.tsx:201]: Somewhat weird to have the button in the payment info component? 🤔

File: src/hooks/kyc.hook.ts [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+export function useKyc(): KycInterface {
+  const { user } = useUserContext();
+
+  const kycMap: Record<string, string> = {
+    ['chatbot']: 'chatbot onboarding',
+    ['onlineid']: 'online identification',
+    ['videoid']: 'video identification',
+    ['check']: 'Data in review',
+    ['manual']: 'Verification completed',
```
Comment (davidleomay) [inline src/hooks/kyc.hook.ts:17]: Manual doesn't exist anymore

File: src/components/tabs/buy-tab-content/buy.process.tsx (hunk 5 above)
Comment (Krysh90) [reply src/components/tabs/buy-tab-content/buy.process.tsx:201]: was on brink to moving it out. will do

File: .vscode/settings.json (hunk 3 above)
Comment (Krysh90) [reply .vscode/settings.json:14]: there are in our stories and i saw warnings therefore I have added them

File: src/components/tabs/buy-tab-content/buy.process.tsx (hunk 2 above)
Comment (Krysh90) [reply src/components/tabs/buy-tab-content/buy.process.tsx:193]: wow yeah. copy paste from story is strong here

**Review by davidleomay — APPROVED** [review]

**Pull request merged**